Changes.md

```diff
@@ -11,7 +11,9 @@ Fixes
 Improvements
 ------------
 
-- Spreadsheet : When promoting plugs - such as those from Attributes node - which have their own "enabled" switch, this switch is now adopted by the spreadsheet, instead of having an additional "enabled" switch on the cell.
+- Spreadsheet :
+  - The enabled/disabled state of a cell may now be edited directly via a new switch in the popup editing window.
```

Comment:
Don't know the right answer here, but do we need to re-assess the rationale in making bool plugs directly editable in the sheet now that we have the enabled switch?  Ie. say we could make other types editable without the hop via the floating widget, would we? I know you almost wanted to get rid of the right-click menu `Disable Cell` item...

https://github.com/GafferHQ/gaffer/blob/8183d7f2bf8e72a587ce994913cbea71e6b6755c/python/GafferUI/SpreadsheetUI.py#L853
https://github.com/GafferHQ/gaffer/blob/8183d7f2bf8e72a587ce994913cbea71e6b6755c/python/GafferUI/SpreadsheetUI.py#L1095

Reply:
I'd be up for getting rid of the special case for bool plugs and preset menus, as well as the `Disable Cell` menu items. I'm not sure how others would feel though, so I think it's best not to bundle it with this change.